packages/i18n/src/locales/en/options/bee.yml

```diff
@@ -72,4 +72,28 @@ neckTieColours:
 
 crossBackTies:
   title: Cross back ties
-  description: Whether you'd like to cross the ties at the back or not
+  description: Whether you'd like to use the cross back tie variation of Bee
+
+bandLength:
```

Comment:
This part surprised me since I couldn't find these options when configuring Bee. Sorry if I am missing something.